testdata/sev_request.json

```diff
@@ -1,6 +1,6 @@
 {
 	"version": "0.0.0",
-    "workload_id": "fakeid",
+        "workload_id": "fakeid",
```

Comment:
For consistency with the other fields, we need to either change this one to make its indentation a tab, or change the others to turn their indentations into spaces.

Reply:
you are right, just fixed